tmt/steps/prepare/install.py

```diff
@@ -199,9 +199,19 @@ def go(self, guest):
 
         # Install from repositories
         if repo_packages:
-            summary = fmf.utils.listed(repo_packages, max=3)
-            self.info('package', summary, 'green')
+            # Show a brief summary by default
+            if not self.opt('verbose'):
+                summary = fmf.utils.listed(repo_packages, max=3)
+                self.info('package', summary, 'green')
+            # Provide a full list of packages in verbose mode
+            else:
+                summary = fmf.utils.listed(repo_packages, 'package')
+                self.info('package', summary + ' requested', 'green')
+                for package in sorted(repo_packages):
+                    self.verbose(package, shift=1)
+            # Quote package names and install
             packages = ' '.join(
-                [tmt.utils.quote(package) for package in packages])
+                [tmt.utils.quote(package) for package in repo_packages])
```

Comment:
What has lead to  this change `packages -> repo_packages`?

Reply:
That was a typo I discovered while working on the patch. Here `repo_packages` should be used as that's the list of packages without the `.rpm` suffix (which is used to detect local rpm files).